alfalfa_worker/jobs/modelica/full_run.py

```diff
@@ -0,0 +1,34 @@
+
+from datetime import datetime
+
+import pytz
+
+from alfalfa_worker.lib.alfalfa_connections_base import AlfalfaConnectionsBase
+from alfalfa_worker.lib.job import Job
+from alfalfa_worker.lib.testcase import TestCase
+
+
+class FullRun(AlfalfaConnectionsBase, Job):
```

Comment:
I think we may have decided to just remove this class since you can do similar with the step run in timescale, though maybe this is not the case. 

If we keep it I'm all for AnnualRun, makes a lot more sense.